crates/c/src/lib.rs

```diff
@@ -3246,5 +3246,41 @@ pub fn to_c_ident(name: &str) -> String {
         "ret" => "ret_".into(),
         "err" => "err_".into(),
         s => s.to_snake_case(),
+    };
+
+    // escape golang keywords because they get used in CGO
```

Comment:
It seems strange to me that the C gen needs to understand what Go keywords are. Could we move this logic to the go gen? Probably what we can do is to sanitize the result of calling `to_c_ident(*)` in the Go gen.

Reply:
I just pushed a commit that swapped the underscore from postfix to prefix to Go keywords. The reason behind this change is that if C struct's field name is a Go keyword, then CGO can access that by prefixing them with the underscore.